### DISCUSSION TITLE
Should a label be created for index-generating markup?

### DISCUSSION THREAD

**Discussion by electric-coder** [sphinx-doc/sphinx#11928] [Q&A]
Does the community think it might be necessary to create a label in the repository for the [index-generating markup][1]?

For example, if we search for the keyword [index][2] the current results return 347 open issues and 1,399 closed issues. But most of these are textual coincidences - here's [one case][3] where the default `index.rst` was included in `conf.py` but nothing in the post is about *index-generating markup* itself.

It's been my experience that cases like these make searching the repository tedious. Historically the repository hasn't kept labels for specific [reST directives][4] but perhaps there might be a substantial gain in time not spent searching, for maintainers and users alike, at the costing of creating more labels.

[1]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#index-generating-markup
[2]: https://github.com/sphinx-doc/sphinx/issues?q=is%3Aissue+is%3Aopen+index
[3]: https://github.com/sphinx-doc/sphinx/issues/11674
[4]: https://github.com/sphinx-doc/sphinx/labels

**Comment by AA-Turner** [comment]
Seems reasonable, do you have a list of issues / PRs you'd include in the label? 

A

**Comment by electric-coder** [comment]
> Seems reasonable, do you have a list of issues / PRs you'd include in the label?

Give me a few days and I'll start going through the repository to compile a list (aiming at sparing maintainers that research time). I'll add it to this thread as an answer.
